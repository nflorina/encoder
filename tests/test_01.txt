8H&GbpNaGIu+!$o[EbUo9JaPko"f|FE5nL
=-3dV*pW/`4TDjE]5'.!v1htW.+
hyxu.^M8P&G1;|V&B4;JC=AT'6-6ATq+
"\JPu{W>.r;dx^y5)=SKrZbp%Ub0%O3bzcXv;w%.
-1181719
>
"
G
L
-13891272
@QQ
L8mCTeIbx3kk
*
y
n
dhJ1jjb=?wjWe/,
oEk[2dfy^c*N.E4voEb[)z!kRfz_3SO
,j%sedWp3e7G]'m@cw
x-2`(qt\BM}O9#C~h{p{b(D@03as+#R%1e
:
1
END
