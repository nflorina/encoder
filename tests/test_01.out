10 8 2
Ln5EF|f"okPaJ9oUb8H&GbpNaGIu+!$o[E+.Wth1v!.'5]EjDT4`=-3dV*pW/+qTA6-6'TA=CJ;4Bhyxu.^M8P&G1;|V&.%w;vXczb3O%0bU%pbZr"\JPu{W>.r;dx^y5)=SK11817191>L1"n1GF1Lf12721389QQ@kk3xbIL8mCTe1*H1yN1n$,/eWjw?=bjdhJ1jOS3_zfRk!z)[bEov4E.N*c^yfd2[kEowc@m']G7e,j%sedWp3e1%R#+sa30@D(b{p{x-2`(qt\BM}O9#C~h1:X
Ln5EF|f"okPaJ9oUb8H&GbpNaGIu+!$o[E+.Wth1v!.'5]EjDT4`=-3dV*pW/+qTA6-6'TA=CJ;4Bhyxu.^M8P&G1;|V&.%w;vXczb3O%0bU%pbZr"\JPu{W>.r;dx^y5)=SK11817191>L1"n1GF1Lf12721389QQ@kk3xbIL8mCTe1*H1yN1n$,/eWjw?=bjdhJ1jOS3_zfRk!z)[bEov4E.N*c^yfd2[kEowc@m']G7e,j%sedWp3e1%R#+sa30@D(b{p{x-2`(qt\BM}O9#C~h1:X
